addons/mail/static/src/components/composer_drop_list_suggestion/composer_drop_list_suggestion.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.ComposerDropListSuggestion" owl="1">
+        <a class="o_ComposerDropListSuggestion dropdown-item" t-att-class="{ 'active': props.isActive }" href="#" role="menuitem" t-on-click="_onClick" t-on-mouseover="_onMouseOver">
+            <t t-if="model">
+                <t t-if="isPartner">
+                    <PartnerImStatusIcon
+                        hasBackground="false"
+                        partnerLocalId="model.localId"
+                    />
+                    <span class="o_ComposerDropListSuggestion_a"><t t-esc="model.nameOrDisplayName"/></span>
+                    <t t-if="model.email">
+                        <span class="o_ComposerDropListSuggestion_b">(<t t-esc="model.email"/>)</span>
+                    </t>
+                </t>
+                <t t-if="isCannedResponse">
+                    <span class="o_ComposerDropListSuggestion_a"><t t-esc="model.source"/></span>
+                    <span class="o_ComposerDropListSuggestion_b"><t t-esc="model.substitution"/></span>
+                </t>
+            </t>
+        </a>
+    </t>
```

Comment:
I would consider having one separate template for each of those, I don't like a big template with plenty of conditions and almost no overlap between them.